monitoring/count-items-cronjob/alerts.yaml

```diff
@@ -0,0 +1,74 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: zenkoCRName
+    type: constant
+    value: artesca-data
+  - name: count_items_cronjob
+    type: constant
+    value: ops-count-items
+  - name: count_items_job_duration_threshold
+    type: config
+    value: 3600
+
+groups:
+  - name: count-items-cronjob/alerts.rules
+    rules:
+      - alert: CountItemsJobTakingTooLong
+        Expr: |
+          (
+            time() - 
+            label_replace(
+              max(
+                kube_job_status_start_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${zenkoCRName}-${count_items_cronjob}"}
+              )
+              BY (job_name, owner_name, namespace)
+              == ON(owner_name) GROUP_LEFT()
+              max(
+                kube_job_status_start_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${zenkoCRName}-${count_items_cronjob}"}
+              )
+              BY (owner_name),
+              "cronjob", "$1", "owner_name", "(.+)")
+          ) > ${count_items_job_duration_threshold}
```

Comment:
This could be simplified by using the metrics `kube_job_status_completion_time` for getting the completion time of a job and `kube_job_status_start_time` to get the start time of a job.

The alert expression should look something like the following:
```
sum by(job_name) (kube_job_status_completion_time{job_name=~"artesca-data-ops-count-items-.*"})
- sum by(job_name) (kube_job_status_start_time{job_name=~"artesca-data-ops-count-items-.*"})
> ${count_items_job_duration_threshold}
```

You can select the group of jobs named `artesca-data-ops-count-items-XXX` using `job_name=~"artesca-data-ops-count-items-.*"`

Reply:
what if a job failed, will it appear in the `kube_job_status_completion_time`?